src/Features/CSharp/Portable/CodeFixes/ImplementAbstractClass/ImplementAbstractClassCodeFixProvider.cs

```diff
@@ -73,7 +73,7 @@ public sealed override FixAllProvider GetFixAllProvider()
         // internal for testing purposes.
         internal static string GetCodeActionId(string assemblyName, string abstractTypeFullyQualifiedName)
         {
-            return CSharpFeaturesResources.ImplementAbstractClass + ";" +
+            return CSharpFeaturesResources.Implement_Abstract_Class + ";" +
```

Comment:
Correct ☺

From: Balaji Krishnan [mailto:notifications@github.com]
Sent: Tuesday, July 19, 2016 11:52 AM
To: dotnet/roslyn <roslyn@noreply.github.com>
Cc: Cyrus Najmabadi <cyrusn@microsoft.com>; Author <author@noreply.github.com>
Subject: Re: [dotnet/roslyn] Remove unnecessary resource. (#12595)


In src/Features/CSharp/Portable/CodeFixes/ImplementAbstractClass/ImplementAbstractClassCodeFixProvider.cs<https://github.com/dotnet/roslyn/pull/12595#discussion_r71400396>:

> @@ -73,7 +73,7 @@ public sealed override FixAllProvider GetFixAllProvider()

>          // internal for testing purposes.

>          internal static string GetCodeActionId(string assemblyName, string abstractTypeFullyQualifiedName)

>          {

> -            return CSharpFeaturesResources.ImplementAbstractClass + ";" +

> +            return CSharpFeaturesResources.Implement_Abstract_Class + ";" +

oh the identifier just matches the text it refers to, including hte casing. So, its the text thats caps on all words, eh?

________________________________

In reply to: 71400121<https://github.com/dotnet/roslyn/pull/12595#discussion_r71400121>

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/dotnet/roslyn/pull/12595/files/0a2f70279c4d0125a51a5751dadff345268ece58#r71400396>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AEWmY6q_Rqk7JuIhsuHoRq2wAArLXhu9ks5qXRzDgaJpZM4JPPlp>.